### PR TITLE
Create an Encounter Viewer

### PR DESCRIPTION
**Pull request by enyachoke**

### CODE REVIEW

**Comment by enyachoke** [comment]
Test Link https://test1.ampath.or.ke:8443/test-builds/AA-588/

File: Gruntfile.js [hunk 1]
```diff
@@ -336,9 +329,9 @@ module.exports = function (grunt) {
     ngtemplates: {
       dist: {
         options: {
-          module: 'openmrsNgresourceApp',
+          module: 'openmrs-ngresource.restServices',
           htmlmin: '<%= htmlmin.dist.options %>',
-          usemin: 'scripts/scripts.js'
+          usemin: 'scripts/openmrsNgresource.js'
```
Comment (mhawila) [inline Gruntfile.js]: Why do you need to change this?

File: Gruntfile.js [hunk 2]
```diff
@@ -336,9 +329,9 @@ module.exports = function (grunt) {
     ngtemplates: {
       dist: {
         options: {
-          module: 'openmrsNgresourceApp',
+          module: 'openmrs-ngresource.restServices',
```
Comment (mhawila) [inline Gruntfile.js]: Is this the name suggested by @jonathandick ?

**Comment by mhawila** [comment]
It looks good except for the comments I have given via slack. Also if there are grouped obs the collapse/expand does not work correctly for subgroups. Try to search for patient with uuid "d0255a53-a8a6-4722-8439-3df4f4c5b1ba" the first encounter (i.e POC Lab encounter of April 11th 2016)

**Pull request merged**